biographyScript.js

```diff
@@ -0,0 +1,67 @@
+let number1 = document.getElementById("number1");
+
+number1.addEventListener("click", function () {
```

Comment:
very creative JS here and throughout the site as well! I really enjoyed clicking through these cards - that's a great testament to your user experience